pkg/installation/move.go

```diff
@@ -243,3 +242,17 @@ func copyFile(source string, dst string, mode os.FileMode) (err error) {
 	}
 	return os.Chmod(dst, mode)
 }
+
+// isCrossDeviceRenameErr determines if a os.Rename error is due to cross-fs/drive/volume copying.
+func isCrossDeviceRenameErr(err error) bool {
+	le, ok := err.(*os.LinkError)
+	if !ok {
+		return false
+	}
+	errno, ok := le.Err.(syscall.Errno)
+	if !ok {
+		return false
+	}
+	return (IsWindows() && errno == 17) || // syscall.ERROR_NOT_SAME_DEVICE
```

Comment:
:+1: for putting the constant names in the comments

Reply:
btw, a search brought this link, kind of same implementation but better documentation

https://github.com/golang/dep/blob/master/internal/fs/rename_windows.go